modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointment.rb

```diff
@@ -267,13 +267,15 @@ def start_date_local
           end
         end
 
+        # rubocop:disable Metrics/MethodLength
```

Comment:
@Tonksthebear there's some linting errors. And please resolve the MethodLength cop here

I'd suggest moving the `when VAOS::V2::AppointmentsService::APPOINTMENT_TYPES[:va]` case to a private method

Reply:
Ah thanks for the suggestion. I was hesitant to break it up to save going one line over but your suggestion is great. Thank you, it should all be resolved now